scripts/ama.coffee

```diff
@@ -60,16 +61,45 @@ class AMAManager
                  ama clear - clears selections (admin only)
                  """
 
+    selectUser = (pass) ->
```

Comment:
pass is there because I was unable to get the function to return a value unless the function took a parameter.  No clue why this is or if there's a better way to do it.